kola/tests/coretest/cloudinit.go

```diff
@@ -0,0 +1,125 @@
+package coretest
+
+import (
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path"
+	"strings"
+	"syscall"
+)
+
+const cloudinitBinPath = "/usr/bin/coreos-cloudinit"
+
+func read(filename string) (string, error) {
+	bytes, err := ioutil.ReadFile(filename)
+	return string(bytes), err
+}
+
+func rmdir(path string) error {
+	cmd := exec.Command("sudo", "rm", "-rf", path)
```

Comment:
if this is mostly reorganization, another pr would probably be fine. it seems there are a lot of instances of exec'ing commands or calling into `syscall` directly, which is suspicious.